src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy

```diff
@@ -1130,6 +1131,119 @@ class LeVADocumentUseCase extends DocGenUseCase {
         return this.createDocument(documentType, repo, data_, [:], modifier, getDocumentTemplateName(documentType, repo), watermarkText)
     }
 
+    /**
+     * Helm releases become top level elements, tailor deployments are left alone.
+     */
+    @SuppressWarnings('CyclomaticComplexity')
```

Comment:
It is not a good practice to suppress quality warnings. Let's talk and see how we could simplify this code.

Reply:
The codeNarc report says its cyclomatic complexity of 21 where the threshold is 20. Now in https://codenarc.org/codenarc-rules-size.html the rules are stated. Adding another statement if (logger.debugMode) to not do json encoding when not logging will raise complexity by one.
Let's discuss whether there are good ideas to make this more clear.